jupyterlab_gitsync/jupyterlab_gitsync/tests/nbmerge_helper_test.py

```diff
@@ -0,0 +1,213 @@
+     "execution_count": null,                                 \n\
+     "metadata": {},                                          \n\
+     "outputs": [],                                           \n\
+     "source": [                                              \n\
+      "print(\\"Hello, this is a cell from the base version of the file.\\")" \n\
+     ]                                                        \n\
+    },                                                        \n\
+    {                                                         \n\
+     "cell_type": "code",                                     \n\
+     "execution_count": null,                                 \n\
+     "metadata": {},                                          \n\
+     "outputs": [],                                           \n\
+     "source": [                                              \n\
+      "print(\\"This is a change from the REMOTE repository.\\")" \n\
+     ]                                                        \n\
+    }                                                         \n\
+   ],                                                         \n\
+   "metadata": {                                              \n\
+    "kernelspec": {                                           \n\
+     "display_name": "",                                      \n\
+     "name": ""                                               \n\
+    },                                                        \n\
+    "language_info": {                                        \n\
+     "name": ""                                               \n\
+    }                                                         \n\
+   },                                                         \n\
+   "nbformat": 4,                                             \n\
+   "nbformat_minor": 4                                        \n\
+  }'
+ 
+local_contents = '{"metadata":{"kernelspec":{"display_name":"","name":""},\
+  "language_info":{"name":""}},"nbformat_minor":4,"nbformat":4,"cells":\
+  [{"cell_type":"code","source":"# This is a change from our LOCAL changes\\n\
+  \\n# This is our base cell\\n# Other cells should be written around this cell\
+  \\n# We can test merge with additions to base cells\\n# and adding cells entirely.",\
+  "metadata":{"trusted":true},"execution_count":null,"outputs":[]},{"cell_type":"code",\
+  "source":"print(\\"This is a cell from our LOCAL changes\\")","metadata":{"trusted":true},\
+  "execution_count":null,"outputs":[]},{"cell_type":"code","source":"print(\\"Hello, this\
+  is a cell from the base version of the file.\\")","metadata":{"trusted":true},\
+  "execution_count":null,"outputs":[]}]}'
+
+def make_setup():
+  subprocess.call(['mkdir', 'test_files'], cwd='.')
+  subprocess.call(['mkdir', 'test_files/.sync_cache'], cwd='.')
+  subprocess.call(['mkdir', 'test_files/.sync_cache/init_cache_test'], cwd='.')
+
+def remove_setup():
```

Comment:
Use the tearDown method instead: https://docs.python.org/3/library/unittest.html#unittest.TestCase.tearDown